packages/astro/src/compiler/transform/styles.ts

```diff
@@ -215,7 +215,7 @@ export default function transformStyles({ compileOptions, filename, fileID }: Tr
                   type: (langAttr && langAttr.value[0] && langAttr.value[0].data) || undefined,
                   filename,
                   scopedClass,
-                  tailwindConfig: compileOptions.tailwindConfig,
+                  tailwindConfig: compileOptions.astroConfig.devOptions.tailwindConfig,
```

Comment:
This makes sense to me. But if we make this change, let’s also remove `tailwindConfig?: string` from the following types:

- [packages/astro/src/@types/compiler.ts#L10](https://github.com/snowpackjs/astro/blob/main/packages/astro/src/@types/compiler.ts#L10)
- [packages/astro/src/compiler/transform/styles.ts#L69](https://github.com/snowpackjs/astro/blob/main/packages/astro/src/compiler/transform/styles.ts#L69)

That way we ensure that this is being passed through as expected.

Reply:
I have removed this from `CompileOptions, but I think you still need it in the `TransformStyleOptions`?

Reply:
Yes that’s fine to keep for now, thanks